cmd/fleet/handleAck.go

```diff
@@ -64,6 +64,7 @@ func (rt Router) handleAcks(w http.ResponseWriter, r *http.Request, ps httproute
 
 		log.WithLevel(resp.Level).
 			Err(err).
+			Str("agentId", id).
```

Comment:
nit: maybe agent_id?

Reply:
Kept to be in alignment with the rest of the app.   Later PR to standardize it.

We have an additional issue that ECS is formatted differently; lower case dot formatted.  But we don't want conflict either, so no good answers.